.gitignore

```diff
@@ -8,20 +8,18 @@ __pycache__/
 coverage.xml
 
 # setuptools
-/build/**/*.*
-/dist/**/*.*
+/build/
+/dist/
 /*.egg-info
 
 # Dependencies
 !requirements.txt
 
 # Sphinx
-doc/_build/
-doc/pyEDAA.ProjectModel/**/*.*
+/doc/_build/
+/doc/_theme/
+/doc/pyEDAA.ProjectModel/
 !doc/pyEDAA.ProjectModel/index.rst
```

Comment:
Why no `!/doc` here?

Reply:
I don't understand. We want to ignore three subdirs `_build`, `_theme` and `pyEDAA.ProjectModel`, but we want to include a file `pyEDAA.ProjectModel/index.rst`.

Reply:
Got it now: the `/` (absolute) was missing.